README.md

```diff
@@ -35,7 +35,7 @@ SQift is a lightweight Swift wrapper for SQLite.
 
 ## Migration Guides
 
-- [SQift 4.0 Migration Guide](Documentation/SQift%204.0%20Migration%20Guide.md)
+- [SQift 4.0 Migration Guide](SQift%204.0%20Migration%20Guide.md)
```

Comment:
Could we go the other way and move the migration guide into a new `Documentation` folder? We'll also want to have a `Documentation` group in the Xcode project with the migration file nested inside it. We obviously missed this when we put that migration guide together.